hypertrace-graphql-gateway-service-metric-utils/src/main/java/org/hypertrace/graphql/utils/metrics/gateway/MetricContainerMapConverter.java

```diff
@@ -87,5 +86,10 @@
     public List<MetricInterval> series(int size, TimeUnit units) {
       return this.metricSeriesMap.get(Duration.of(size, units.getChronoUnit()));
     }
+
+    @Override
+    public List<BaselineMetricInterval> baselineSeries(int size, TimeUnit units) {
+      return null;
```

Comment:
Never return null, please have this return an empty list with a comment that it's not implemented.